### ISSUE TITLE
Fatal error: Uncaught Casper\Rpc\RpcError: invalid deploy: the provided hash does not match the actual hash of the deploy

### ISSUE THREAD

**Issue by Jiuhong-casperlabs** [make-software/casper-php-sdk#14]
**What happened?**
Hi, I am new to PHP. I tested the transfer code in [SendingTransfe](https://github.com/make-software/casper-php-sdk/blob/master/docs/Example/SendingTransfer.md) but it gave me the following error.  The code is attached in the comment. 
```
Fatal error: Uncaught Casper\Rpc\RpcError: invalid deploy: the provided hash does not match the actual hash of the deploy in C:\xampp\htdocs\myproject1\composer_1\vendor\make-software\casper-php-sdk\src\Rpc\RpcClient.php:380 Stack trace: #0 C:\xampp\htdocs\myproject1\composer_1\vendor\make-software\casper-php-sdk\src\Rpc\RpcClient.php(78): Casper\Rpc\RpcClient->rpcCallMethod('account_put_dep...', Array) #1 C:\xampp\htdocs\myproject1\composer_1\06_contracthash.php(33): Casper\Rpc\RpcClient->putDeploy(Object(Casper\Entity\Deploy)) #2 {main} thrown in C:\xampp\htdocs\myproject1\composer_1\vendor\make-software\casper-php-sdk\src\Rpc\RpcClient.php on line 380
```

**What did you expect to happen?**
No error happens.

**Are you willing to submit a pull request to fix this bug?**
No.

**Comment by Jiuhong-casperlabs** [comment]
This is the entire code I tested. 
```php
<?php
// NG
require __DIR__ . '/vendor/autoload.php';

$nodeAddress = 'http://65.21.237.160:7777';
$rpcClient = new Casper\Rpc\RpcClient($nodeAddress);

// Replace '/path/to/secp256k1_secret_key.pem' by real path to secret key
$senderKeyPair = Casper\Util\Crypto\Secp256K1Key::createFromPrivateKeyFile('./key/secret_key_2.pem');
$senderPublicKey = Casper\Serializer\CLPublicKeySerializer::fromAsymmetricKey($senderKeyPair);

// Replace 'recipient_hex_public_key_here' by real public key
$recipientPublicKey = Casper\Serializer\CLPublicKeySerializer::fromHex('02032f2eac2b4b5d5a4129612be91aca6a0adaf8e5785719519d8c7bcf6d0617a997');

$transferId = 1;
$transferAmount = 2500000000;
$transfer = Casper\Entity\DeployExecutable::newTransfer($transferId, $transferAmount, $recipientPublicKey);

$paymentAmount = 100000000;
$payment = Casper\Entity\DeployExecutable::newStandardPayment($paymentAmount);

$networkName = 'casper-test';
$deployParams = new Casper\Entity\DeployParams($senderPublicKey, $networkName);

$deployService = new Casper\Service\DeployService();
$deploy = $deployService->makeDeploy($deployParams, $transfer, $payment);

$signedDeploy = $deployService->signDeploy($deploy, $senderKeyPair);
$deployHash = $rpcClient->putDeploy($signedDeploy);
?>
```

**Comment by RomanovSci** [comment]
Fixed with release 1.2.0
